frontend/compositions/state-select/state-select.tsx

```diff
@@ -1,38 +1,41 @@
 import React, { FormEvent, useState } from "react"
 import styles from "./state-select.module.css"
-import { FormLevelError } from '../index'
-import { states } from '../../models'
+import { FormLevelError } from "../../shared-components/index"
+import { states } from "../../models"
 
-interface USStateSelectProps { isSubmitted: boolean }
+interface USStateSelectProps {
+  isSubmitted: boolean
+}
 export default function USStateSelect({ isSubmitted }: USStateSelectProps) {
-  const [selectId, errorId] = ['stateSelect', 'stateSelectError']
+  const [selectId, errorId] = ["stateSelect", "stateSelectError"]
```

Comment:
I don't understand this particular code style: if the variables are strings, why not just declare them as strings?

Reply:
ya, this is just kinda a slick way to do it that will allow for expanding on later but my only change was the quotes and that was due to the linting not me by choice. It does work so I didn't question it at the time but may @srsexton94 can elaborate on the declaration here?